sl_model_mod.f90

```diff
@@ -1296,6 +1273,20 @@ subroutine sl_solver(itersl, iter, dtime, starttime, mali_iceload, mali_mask, sl
       write(201,'(I4)') nmelt
       close(201)
 
+      ! number of iteration it takes for the inner convergence
+      open(unit = 201, file = trim(outputfolder)//'numiter', form = 'formatted', access ='sequential', &
+      & status = 'old', position='append')
```

Comment:
Will this file have the same issue - that if a restart causes some time steps to be rerun there will be multiple copies of those lines?  Maybe that doesn't matter much for this file but wanted to point it out.

Reply:
@matthewhoffman 
Yes - this file will have the same issue, along with other files like `times` as well. These two files won't affect the SLM calculation itself but affect post-processing if used. But those files are rarely used nor looked at. So maybe I can either just stop writing those into a file in the first place, or we just keep it as is for now and address this issue in a new PR. The new PR could be even on making all the I/O to happen in the netcdf-format (because it's been proven that the text-formatted files take up way too much space (as we've recently seen in my project directory on Chicoma), and the remaining issue will be automatically taken care of. What do you think?

Reply:
I'm fine with that limitation with this file for now if you are.  I don't imagine I will be looking at them, so if you don't think they are critical, that's ok with me.